src/ssWeatherManager.lua

```diff
@@ -155,6 +155,10 @@ function ssWeatherManager:readStream(streamId, connection)
 
         table.insert(self.rains, rain)
     end
+
+    -- we need to synch this, we have an error log entry on dedicated server otherwise
+    -- for now i do it here, it's not elegant but at the moment it work
+    ssSnow.appliedSnowDepth = streamReadFloat32(streamId)

```

Comment:
That is already the case: on the client the hour change listener is never installed